src/components/Calculator/FormItem/styles.module.css

```
@@ -18,15 +18,28 @@
 
 .formItem input,
 .formItem select {
+  -webkit-appearance: none; /* 移除 iOS Safari 默认 UI */
```

Let's use English in comments

```suggestion
  -webkit-appearance: none; /* Remove the default UI in iOS Safari */
```